client/src/game/ui/tools/DiceTool.vue

```diff
@@ -11,20 +13,52 @@ import { DxHelper } from "../../systems/dice/dx";
 import { diceState } from "../../systems/dice/state";
 import { diceTool } from "../../tools/variants/dice";
 
-const showHistory = ref(false);
+const { t } = useI18n();
+
+// const limitOperatorOptionNames = [t('game.ui.tools.DiceTool.limit_operator_options.keep'), t('game.ui.tools.DiceTool.limit_operator_options.drop'), t('game.ui.tools.DiceTool.limit_operator_options.min'), t('game.ui.tools.DiceTool.limit_operator_options.max')]
+// const selectorOptionNames = ["=", ">", "<", t('game.ui.tools.DiceTool.selection_option_names.highest'), t('game.ui.tools.DiceTool.selection_option_names.lowest')]
```

Comment:
What's the deal with these?

Reply:
These should be the i18n names in ClickGroup. I commented instead of removing them since you have said there would be an update on it to have the same API as ToggleGroup (like `{label: string; value: T}`) and they could be put into use then.